### PR TITLE
First Milestone: Blog app - creating a data model

### PR DESCRIPTION
**Pull request by digia3d**
In this milestone I did the following:

- Make sure that my project has a Postgres database set up.
- Build my project schema:

1.     Create and run the necessary migration files.
2.     Table and column names match the ERD diagram.
3.     Foreign keys included.
4.     All columns that are foreign keys have a corresponding index.

### CODE REVIEW

File: db/migrate/20221019123217_add_user_to_posts.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :author, null: false, foreign_key: { to_table: :users }
```
Comment (DammyShittu) [inline db/migrate/20221019123217_add_user_to_posts.rb:3]: Good job adding references for the posts to the author through the users' table.

- [x] Kindly add the same references to the author as well for the likes table.

- [x] Kindly add the same references to the author as well for the comments table

File: README.md [hunk 2]
```diff
@@ -1 +1,24 @@
-# Blog_App
\ No newline at end of file
+# README
```
Comment (DammyShittu) [inline README.md]: - [ ] Kindly update your readme with the template from Microverse

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @digia3d,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (DammyShittu) [inline README.md]: - [ ] Kindly update your readme with the template from Microverse as requested earlier

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @digia3d,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @digia3d, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**